Sources/Jetworking/Client/Client.swift

```diff
@@ -90,15 +91,17 @@ public final class Client {
 
     // MARK: - Methods
     @discardableResult
-    public func get<ResponseType>(endpoint: Endpoint<ResponseType>, _ completion: @escaping RequestCompletion<ResponseType>) -> CancellableRequest? {
+    public func get<ResponseType: Decodable>(endpoint: Endpoint<ResponseType>, _ completion: @escaping RequestCompletion<ResponseType>) -> CancellableRequest? {
         do {
             let request: URLRequest = try createRequest(forHttpMethod: .GET, and: endpoint)
             return requestExecutor.send(request: request) { [weak self] data, urlResponse, error in
-                self?.handleResponse(
+                guard let self = self else { return }
```

Comment:
Please add an empty line after the `guard`.

Reply:
Done